benchmarks/matio_matio_fuzzer/Dockerfile

```diff
@@ -21,3 +21,5 @@ RUN git clone --depth 1 git://git.code.sf.net/p/matio/matio matio
 WORKDIR matio
 COPY build.sh $SRC/
 COPY *.cc *.h $SRC/
+COPY testcases/ $SRC/
+COPY testcases/ $SRC/
```

Comment:
Is this second copy needed?

Reply:
It shouldn't even be there yet, it's part of the testing I was doing on another pull request I am still working on. I will remove both of them since they are still not needed at the moment.
Thanks for noticing that